config/initializers/bugsnag.rb

```diff
@@ -3,9 +3,9 @@
 Bugsnag.configure do |config|
   config.api_key = Rails.configuration.three_scale.bugsnag_api_key
   config.app_version = System::Deploy.info.revision
-  config.notify_release_stages = %w[production]
   stages = Rails.configuration.three_scale.error_reporting_stages
-  config.notify_release_stages = stages if stages.present?
+  config.notify_release_stages = stages.present? ? stages : %w[production]
```

Comment:
if we are not notified about errors, how would we know about them?

Reply:
We would get notified! Just need to set `error_reporting_stages` in `settings.yml` properly.

Example: https://github.com/3scale/3scale-saas/pull/449/files#diff-5209d018920a63ac8777bf0474e3f0ee84268d2f7cb1a51b0a5cd9bbef878b70R20

Reply:
why not add `staging` by default?

Reply:
Well, `staging` is an environment that only exists in SaaS world, so I don't know if there would be much benefit in adding it here. But we can do it also.